test/python/transpiler/test_coupling.py

```diff
@@ -448,6 +448,16 @@ def test_implements_iter(self):
         expected = [(0, 1), (1, 0), (1, 2), (2, 1)]
         self.assertEqual(sorted(coupling), expected)
 
+    def test_equality(self):
+        """Test that equality is based on graph isomorphism."""
+
+        coupling0 = CouplingMap([(0, 1), (0, 2), (2, 3)])
+        coupling1 = CouplingMap([(0, 1), (0, 2), (2, 3)])
+        coupling2 = CouplingMap([(0, 1), (0, 2), (2, 4)])
```

Comment:
I think maybe having a comment here to say this will create a 5 node coupling map with 3 disconnected (it might not be obvious from a quick glance).